main.py

```
@@ -0,0 +1,245 @@
+logger = logging.getLogger(__name__)
+handler = logging.StreamHandler(sys.stdout)
+logger.addHandler(handler)
+TABLE_NAMES = declare_constants.GET_TABLE_NAMES()
+
+app = FastAPI()
+
+# Mount static files directory
+app.mount("/static", StaticFiles(directory="static"), name="static")
+
+@app.get("/")
+async def index(request: Request):
```

Ruff (0.8.2)

Missing return type annotation for public function `index`

(ANN201)